### DISCUSSION TITLE
Is  DIsk free supposed to include trash?

### DISCUSSION THREAD

**Discussion by sgoldthorpe** [anclrii/Storj-Exporter#48] [Q&A]
On my node I have just under 500MB free - I'm getting this in Grafana.  Should Disk free be including trash?
Disk total: 1.900 TB
Disk used: 1.895 TB
Disk free: 4.520 GB
Disk trash: 4.020 GB

**Comment by anclrii** [comment]
I calculate `disk-free` as `disk-total - disk-used` where `disk-used` is populated from this api value https://github.com/storj/storj/blob/54dd430048a2bdbd940f31c1787dfbe8cf51d454/storagenode/pieces/store.go#L578-L592

> // SpaceUsedForPiecesAndTrash returns the total space used by both active
> // pieces and the trash directory.
> ...

So disk-free should not include trash. Looking at your grafana numbers you should have `4.520 GB` actually free on disk (unless value changed in api). There are a number of things that can lead to disk space being misrepresented on the node. If you can check how much disk space is actually used by the node on the filesystem does it match `Disk used` value? Also what OS are you running?
